### PR TITLE
Added the FabricBot configuration file

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
Fabric Bot now supports code-based configuration. That means, as soon as this change is merged, Fabric Bot will be using this configuration going forward, instead of the one in the portal.

For now the values in this file can't reference any variables known to the build system - Version.props.
But I've reached out to the FabricBot team and they've agreed that the scenario makes sense and added [an item to their backlog](https://mseng.visualstudio.com/1ES/_workitems/edit/1919391) to support this in the future.

Also, for now there is no support for per-branch configuration, but support for it is also coming. They're already working on that.

This change also impacts the branching instructions. PR for the change is here: https://github.com/dotnet/aspnetcore-internal/pull/4010

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]
Nice! We should do some follow up items - update https://github.com/dotnet/aspnetcore-internal/blob/main/.github/ISSUE_TEMPLATE/ReleasePreparation-preview.md to reference this instead of the UI, and add a small doc describing how to make changes to the config. Doug or I will handle the first, let us know if you want us to take the second too

File: .github/fabricbot.json [hunk 1]
```diff
@@ -0,0 +1,3274 @@
...
+      "eventNames": [
+        "pull_request",
+        "issues",
+        "project_card"
+      ],
+      "actions": [
+        {
+          "name": "addMilestone",
+          "parameters": {
+            "milestoneName": "7.0-preview2"
```
Comment (dougbu) [inline .github/fabricbot.json]: Can we make this
```suggestion
            "milestoneName": "7.0-preview3"
```
and merge after I branch tonight❔ Or should we get this in quick then follow up almost immediately w/ a 'main'-only PR to make this change❔

**Review by dougbu — COMMENTED** [review]
I have no way of knowing if this file is corrupted or not. @wtgodbe what did you base your approval on 😁

**Comment by wtgodbe** [comment]
> I have no way of knowing if this file is corrupted or not. @wtgodbe what did you base your approval on 😁

The file is auto-generated by fabric bot, I assumed it knew what it was doing 😄

**Comment by mkArtakMSFT** [comment]
> > I have no way of knowing if this file is corrupted or not. @wtgodbe what did you base your approval on 😁
> 
> The file is auto-generated by fabric bot, I assumed it knew what it was doing 😄

Correct. I've simply exported the rules from FabricBot. However, to make sure we don't have any random rules flying around, I'm going to compare this with the one from the aspnetcore-internal repo.

**Comment by mkArtakMSFT** [comment]
And it looks like the file is in good shape. Here is the only change I see:
image

**Comment by dougbu** [comment]
@mkArtakMSFT I just updated the bot to add new 'main' commits to preview**3**. Please change that line here to avoid going backward when you merge.

File: .github/fabricbot.json (hunk 1 above)
Comment (dougbu) [inline .github/fabricbot.json]:
```suggestion
            "milestoneName": "7.0-preview3"
```

Comment (dougbu) [reply .github/fabricbot.json]: I don't think I'm looking at the latest version but also see no indication of a new commit taking this change. I keep making the suggestion and then am unable to commit it.

**Comment by dougbu** [comment]
Thanks @mkArtakMSFT I was getting so confused

**Comment by mkArtakMSFT** [comment]
Linked the instructions update PR in the description.

**Pull request merged**